src/Kinepolis/KinepolisService.php

```diff
@@ -72,7 +74,8 @@ public function __construct(
         UuidGeneratorInterface $uuidGenerator,
         TrailerRepository $trailerRepository,
         ProductionRepository $productionRepository,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        bool $trailersEnabled = true
```

Comment:
I'm not a fan of default parameters. It's better to put the default behavior at the place where the constructor is called.